_posts/institute-materials/day1_monday/2016-06-20-openNEONH5.md

```diff
@@ -1,26 +1,33 @@
 ---
```

Comment:
I'm really confused as to why this is showing up as changed.  I haven't modified the YAML at all.